packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc

```diff
@@ -117,7 +120,10 @@ class ConnectivityPlusTizenPlugin : public flutter::Plugin {
       Connection connection;
       ConnectionType type = connection.GetType();
       if (type != ConnectionType::kError) {
-        result->Success(flutter::EncodableValue(ConnectionTypeToString(type)));
+        flutter::EncodableList encodedList;
```

Comment:
Suggest changing  encodedList to encoded_list

Reply:
Thank you. I updated PR